packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewOutputList/TransactionReviewOutputList.tsx

```diff
@@ -147,6 +147,7 @@ export const TransactionReviewOutputList = ({
                 bitcoinLockTime: !!options.includes('bitcoinLockTime'),
                 ethereumData: !!options.includes('ethereumData'),
                 rippleDestinationTag: !!options.includes('rippleDestinationTag'),
+                stellarMemo: !!options.includes('stellarMemo'),
```

Comment:
we have special logic for `rippleDestinationTag`, would be nice to use it also for `stellarMemo`

Reply:
We have refactored this part, and now Stellar and Ripple reuse this code.

https://github.com/trezor/trezor-suite/pull/16483/files#diff-fd866cc0695afb3249f717224024fc84bcb5b462926eab976666d8acf0a514cbR23